bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/installcomponents/container/InstallComponentsCompositePage.java

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+ * Copyright 2021 Espressif Systems (Shanghai) PTE LTD. All rights reserved.
+ * Use is subject to license terms.
+ *******************************************************************************/
+package com.espressif.idf.ui.installcomponents.container;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.core.resources.IFile;
+import org.eclipse.swt.widgets.Composite;
+
+import com.espressif.idf.ui.installcomponents.desrializer.ComponentsDesrializer;
```

Comment:
correction `desrializer` to `deserializer`?

Reply:
If possible try to avoid too many packages unless it's required

Reply:
Done refactored properly